packages/gamut/src/Form/Select.tsx

```diff
@@ -5,28 +5,32 @@ import {
 import { variant } from '@codecademy/gamut-styles';
 import { css } from '@emotion/react';
 import styled from '@emotion/styled';
-import { each, isArray, isObject } from 'lodash';
 import React, {
   ChangeEvent,
   forwardRef,
-  ReactNode,
   SelectHTMLAttributes,
+  useMemo,
   useState,
 } from 'react';
 
 import { Box, FlexBox } from '../Box';
 import { conditionalStyles, formFieldStyles } from './styles/shared';
+import { parseOptions } from './utils';
 
-export type SelectWrapperBaseProps = SelectHTMLAttributes<HTMLSelectElement> & {
+export type SelectComponentProps = Pick<
+  SelectHTMLAttributes<HTMLSelectElement>,
+  'disabled'
+> & {
   error?: boolean;
   htmlFor?: string;
   options?: string[] | Record<string, number | string>;
   id?: string;
 };
 
-export type SelectWrapperProps = SelectWrapperBaseProps & {
-  sizeVariant?: 'small' | 'base';
-};
+export type SelectWrapperProps = SelectComponentProps &
+  SelectHTMLAttributes<HTMLSelectElement> & {
```

Comment:
If `SelectComponentProps` is a `Pick` from `SelectHTMLAttributes<HTMLSelectElement> ` already...

```suggestion
export type SelectWrapperProps = SelectHTMLAttributes<HTMLSelectElement> & {
```

Reply:
it's separate because i export + reuse `SelectComponentProps` in `SelectDropdown`